hazelcast/cluster.py

```diff
@@ -284,6 +287,17 @@ def _members_string(members):
     def _create_snapshot(version, member_infos):
         new_members = OrderedDict()
         for member_info in member_infos:
+            address_map = member_info.address_map
+            if address_map:
+                address = address_map.get(
+                    _CLIENT_ENDPOINT_QUALIFIER,
+                    address_map.get(_MEMBER_ENDPOINT_QUALIFIER, member_info.address),
```

Comment:
Isn't this second default value `member_info.address` extra compared to java implementation? In Java if address_map.get(_MEMBER_ENDPOINT_QUALIFIER) will return null, address of member will be null. 

```java
memberBuilder = new MemberImpl.Builder(addressMap)
                        .address(addressMap.getOrDefault(CLIENT, addressMap.get(MEMBER)));
```

Reply:
I don't like the behaviour but anyways, I reverted my addition